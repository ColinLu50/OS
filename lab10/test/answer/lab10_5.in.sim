2914
317
405
330
317
278
