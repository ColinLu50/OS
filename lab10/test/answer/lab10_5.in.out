FCFS
70 -> 195 -> 170 -> 67 -> 335 -> 264 -> 326 -> 259 -> 53 -> 300 -> 92 -> 44 -> 231 -> 157 -> 53 -> 208 -> 237 -> 85 -> 133 -> 83 -> 126 -> 135 -> 325 -> 103 -> 289 -> 254
2914
SSTF
70 -> 67 -> 53 -> 53 -> 44 -> 83 -> 85 -> 92 -> 103 -> 126 -> 133 -> 135 -> 157 -> 170 -> 195 -> 208 -> 231 -> 237 -> 254 -> 259 -> 264 -> 289 -> 300 -> 325 -> 326 -> 335
317
SCAN
70 -> 67 -> 53 -> 44 -> 0 -> 83 -> 85 -> 92 -> 103 -> 126 -> 133 -> 135 -> 157 -> 170 -> 195 -> 208 -> 231 -> 237 -> 254 -> 259 -> 264 -> 289 -> 300 -> 325 -> 326 -> 335
405
C-SCAN
70 -> 67 -> 53 -> 44 -> 0 -> 343 -> 335 -> 326 -> 325 -> 300 -> 289 -> 264 -> 259 -> 254 -> 237 -> 231 -> 208 -> 195 -> 170 -> 157 -> 135 -> 133 -> 126 -> 103 -> 92 -> 85 -> 83
330
LOOK
70 -> 67 -> 53 -> 44 -> 83 -> 85 -> 92 -> 103 -> 126 -> 133 -> 135 -> 157 -> 170 -> 195 -> 208 -> 231 -> 237 -> 254 -> 259 -> 264 -> 289 -> 300 -> 325 -> 326 -> 335
317
C-LOOK
70 -> 67 -> 53 -> 44 -> 335 -> 326 -> 325 -> 300 -> 289 -> 264 -> 259 -> 254 -> 237 -> 231 -> 208 -> 195 -> 170 -> 157 -> 135 -> 133 -> 126 -> 103 -> 92 -> 85 -> 83
278
